src/main/java/com/study/realworld/secure/IGenerator.java

```diff
@@ -0,0 +1,9 @@
+package com.study.realworld.secure;
+
+public interface IGenerator {
+	static final String _upper = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+	static final String _upperHex = "0123456789ABCDEF";
```

Comment:
```suggestion
	static final String UPPER = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ";
	static final String UPPER_HEX = "0123456789ABCDEF";
```
상수에 대해서 자바 컨벤션은 항상 대문자 입니다 :)

Reply:
바뀐 상수들에 대해서 런 이슈있었던 부분들 새로 수정하여 커밋했습니다!